manim/mobject/text/text_mobject.py

```diff
@@ -197,15 +179,24 @@ def _gen_chars(self, lines_str_list):
         char_index_counter = 0
         chars = self.get_group_class()()
         for line_no in range(lines_str_list.__len__()):
```

Comment:
While we are at it... 😅 

```suggestion
        for line_no in range(len(lines_str_list)):
```

Reply:
😅